lib/app.js

```diff
@@ -1,11 +1,16 @@
 const express = require('express');
-
+const path = require('path');
 const app = express();
 
 // Built in middleware
 app.use(express.json());
-
+app.use(express.static(path.join(__dirname, 'public')));
 // App routes
+app.use('/people', require('./controllers/people'));
+app.use('/dog', require('./controllers/dog'));
+app.use('/overwatch', require('./controllers/overwatch'));
+app.use('/shoes', require('./controllers/shoe'));
+app.use('/cat', require('./controllers/cat'));
```

Comment:
In the future, I would make all of you controllers/routes plural, to fit how they are conventionally named.